ClearKeep/Scenes/PeerCall/PeerCallView.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  PeerCallView.swift
+//  ClearKeep
+//
+//  Created by Luong Minh Hiep on 18/05/2021.
+//
+
+import SwiftUI
+import CommonUI
+import ChatSecure
+import Combine
+import Common
+import Model
+
+struct PeerCallView: View {
+	@ObservedObject var viewModel: CallViewModel
+	// MARK: - Variables
+	@Environment(\.injected) private var injected: DIContainer
+	@State private var isLoading: Bool = false
+	@State private var isShowError: Bool = false
+	@State private var alertType: InCallView.AlertCall = .changeTypeCall
+	
+	@State private(set) var loadable: Loadable<IPeerViewModels> = .notRequested {
+		didSet {
+			switch loadable {
+			case .loaded(let load):
+				isLoading = false
+				if let changeCallVideo = load.callVideoModel {
+					if changeCallVideo.error.isEmptyOrNil {
+						self.viewModel.updateCallTypeVideo()
+					}
+				}
+			case .failed(let error):
+				isLoading = false
+				self.alertType = .error(error: LoginViewError(error))
+				self.isShowError = true
+			case .isLoading:
+				isLoading = true
+			default: break
+			}
+		}
+	}
+	
+	var body: some View {
+		GeometryReader { reader in
+			ZStack(alignment: .top) {
+				if let avatar = viewModel.callBox?.avatar {
+					AsyncImage(url: URL(string: avatar)) { image in
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/fe9f474e9dacdb67166a3b18715e917932515a0f/ClearKeep/Scenes/PeerCall/PeerCallView.swift#L48"></span>Arguments should be either on the same line, or one per line.<br />
<code>multiline_arguments</code> <code>PeerCallView.swift:48</code></td>
    </tr>
  </tbody>
</table>